flow360/component/simulation/outputs/outputs.py

```diff
@@ -33,7 +33,38 @@
 
 
 class UserDefinedField(Flow360BaseModel):
-    """Defines additional fields that can be used as output variables"""
+    """
+
+    Defines additional fields that can be used as output variables.
+
+
+    Example
+    -------
+
+    - Compute :code:`Mach` using :class:`UserDefinedField`
+      (Showcase use, already supported in :ref:`Output Fields <UniversalVariablesV2>`):
+
+    >>> fl.UserDefinedField(
+    ...     name="Mach_UDF",
+    ...     expression="double Mach = sqrt(primitiveVars[1] * primitiveVars[1] + "
+    ...     + "primitiveVars[2] * primitiveVars[2] + primitiveVars[3] * primitiveVars[3])"
+    ...     + " / sqrt(gamma * primitiveVars[4] / primitiveVars[0]);",
+    ... )
+
+
+    - Compute :code:`PressureForce` using :class:`UserDefinedField`:
+
+    >>> fl.UserDefinedField(
+    ...     name="PressureForce",
+    ...     expression="double p = primitiveVars[4] - pressureFreestream; "
+    ...     + "PressureForce[0] = p * rotationModel_nodeNormals[0]; "
```

Comment:
Are you sure `rotationModel_nodeNormals` is correct?

Reply:
Fixed